assets/agenda/components/AgendaListItem.tsx

```diff
@@ -136,6 +136,15 @@ class AgendaListItem extends React.Component<any, any> {
         // Show headline for adhoc planning items
         const showHeadline = !item.event && get(item, 'headline.length', 0) > 0;
 
+        const descriptionHTML = description.replace('\n', '<br/>').replace('<br/>', '</p>\n<p>');
+
+        function isHTML(value: string) {
+            const doc = new DOMParser().parseFromString(value, 'text/html');
+            return Array.from(doc.body.childNodes).some(node => node.nodeType === 1);
+        }
+
+        const descriptionArray = isHTML(description) ? descriptionHTML : description;    
```

Comment:
not sure about the naming there, because neither `descriptionHTML` nor `description` is an array